modules/core/src/core/transform.js

```diff
@@ -80,7 +80,7 @@ export default class Transform {
   // Return Buffer object for given varying name.
   getBuffer(varyingName = null) {
     assert(varyingName && this.feedbackBuffers[this.currentIndex][varyingName]);
-    return this.feedbackBuffers[this.currentIndex][varyingName];
+    return this.feedbackBuffers[this.currentIndex][varyingName].buffer || this.feedbackBuffers[this.currentIndex][varyingName];
```

Comment:
Explicitly check if the object is an instance of Buffer? If someone accidentally add a `buffer` field to the class this will be broken.